### PR TITLE
chore(docs): update release management process and docs

### PR DESCRIPTION
**Pull request by kmova**
Add a roadmap doc to provide an overview of useful github trackers.
Update the Release management process doc with monthly release cadence process
Include additional information about the "status" of data engines.

Signed-off-by: kmova <kiran.mova@mayadata.io>

### CODE REVIEW

File: contribute/process/release-management.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# OpenEBS Release Management
+
+OpenEBS follows a monthly release cadence. The scope of the release is determined by contributor availability and the pending items listed in the [roadmap](../../ROADMAP.md). The scope is published in the [Release Tracker Projects](https://github.com/openebs/openebs/projects). 
+
+## Overview
+
+* Release Manager is identified at the begining of each release from the contributor community, who will work with one of the maintainers of the OpenEBS project. Release manager is responsible for tracking the scope, co-ordinating with various stack holders and helping root out the risks to the release as early as possible. Release manager uses the [Daily Standup](https://github.com/openebs/openebs/tree/master/community#regular-standup-and-release-cadence-meetings) and the [Release Tracker](https://github.com/openebs/openebs/projects) to help everyone stay focused on the release.  
+* Each release has the following important milestones that have organically developed to help maintain the montly release cadence.
```
Comment (gprasath) [inline contribute/process/release-management.md]: monthly

File: contribute/process/release-management.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
+# OpenEBS Release Management
+
+OpenEBS follows a monthly release cadence. The scope of the release is determined by contributor availability and the pending items listed in the [roadmap](../../ROADMAP.md). The scope is published in the [Release Tracker Projects](https://github.com/openebs/openebs/projects). 
+
+## Overview
+
+* Release Manager is identified at the begining of each release from the contributor community, who will work with one of the maintainers of the OpenEBS project. Release manager is responsible for tracking the scope, co-ordinating with various stack holders and helping root out the risks to the release as early as possible. Release manager uses the [Daily Standup](https://github.com/openebs/openebs/tree/master/community#regular-standup-and-release-cadence-meetings) and the [Release Tracker](https://github.com/openebs/openebs/projects) to help everyone stay focused on the release.
```
Comment (ChandanSagar) [inline contribute/process/release-management.md]: > at the begining of each release from

begining -->beginning

File: contribute/process/release-management.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+  * Second Week - Depending on the feature status - alpha, beta or stable, the various tasks from development, review, e2e and docs are identified. Any blockers in terms of design or implementation are discussed and mitigated.
+  * Third Week - Release branch is created and the first Release candidate build is made available. Post this stage only bug fixes are accepted into the release. Upgrades are tested. Staging documentation is updated. CI Pipelines are automated with new tests. 
+  * Fourth Week - Release notes are prepared. The final RC build is pushed to Dogfooding and beta testing with some of the users. The installers are updated. Release, Contributor and User Documentation are published. 
+  * Fifth Week - Post release activities like pushing the new release to the partner charts like Rancher Catelog, Amazon marketplace and so forth are worked on.
```
Comment (ChandanSagar) [inline contribute/process/release-management.md]: >like Rancher Catelog, Amazon marketplace

Catelog --> Catalog

File: contribute/process/release-management.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+Once all the above tests are completed, a main release tagged images, helm and operator YAMLs are published.
+
+## Release Tagging
+
+OpenEBS Components are released as container images with versioned tags. 
+
+OpenEBS components are spread across various repositories. Creating a release tag on the repository will trigger the build, integration tests and pushing of the docker image to [docker hub](https://hub.docker.com/u/openebs) and [quay](https://quay.io/organization/openebs/) container repositories. 
+
+The format of the release tag is either "Release-Name-RC1" or "Release-Name" depending on whether the tag is a release candidate or a release. (Example: 1.0.0-RC1 is a github release tag for openebs release candidate build. 1.0.0 is the release tag that is created after the release criteria is statisfied by the release candidate builds.)
```
Comment (ChandanSagar) [inline contribute/process/release-management.md]: > criteria is statisfied by the

statisfied  --> satisfied

File: contribute/process/release-management.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+
+Each of the respository has the automation scripts setup to push in the container images to docker and quay container repositories.
```
Comment (ChandanSagar) [inline contribute/process/release-management.md]: > Each of the respository has

respository --> repository

File: contribute/process/release-management.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+- There are no release blockers identified in the [Release Candidate Verification Checklist](#release-candidate-verification-checklist) on the final release tagged images.
+- The updated Install and Feature documentation are verified. 
+- Release notes with changes summary, changelog are updated. 
+- openebs-operator and helm charts are published.
+- Release is tagged on openebs/openebs repository.
+- Release is announced on slack, distribution lists and social media
+
+## Post Release Activites
+- Blogs on new features are published
+- Update the news letter contents
```
Comment (ChandanSagar) [inline contribute/process/release-management.md]: > Update the news letter contents

news letter  --> newsletter

File: ROADMAP.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
+# Roadmap
+
+This document provides information on OpenEBS development in current and upcoming releases. Community and contributor involvement is vital for successfully implementing all desired items for each release. We hope that the items listed below will inspire further engagement from the community to keep OpenEBS progressing and shipping exciting and valuable features.
+
+## OpenEBS Lean Roadmap
+
+OpenEBS follows lean project management approach by splitting the development items into current, near term and future categories. We use GitHub [Projects](https://github.com/orgs/openebs/projects) and [Milestones](https://github.com/openebs/openebs/milestones) for tracking the feature developement. This document is reviewed and updated based in the [monthly community call](https://github.com/openebs/openebs/tree/master/community#regular-monthly-product-meetings).
```
Comment (ChandanSagar) [inline ROADMAP.md]: > OpenEBS follows lean project management

OpenEBS follows a lean project management

Comment (ChandanSagar) [reply ROADMAP.md]: > reviewed and updated based in the

in --> on

**Comment by kmova** [comment]
Thank you @gprasath @ChandanSagar for the reviews. Incorporated the corrections.

File: contribute/process/release-management.md [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+- openebs/cstor
+- openebs/istgt
+- openebs/jiva
+- openebs/node-disk-manager
+- openebs/external-storage
+- openebs/velero-plugin
+- openebs/maya
+- openebs/cstor-csi
+- openebs/zfs-localpv
+
```
Comment (ChandanSagar) [inline contribute/process/release-management.md]: openebs/jiva-csi repo should be added here.

File: ROADMAP.md [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+
+This document provides information on OpenEBS development in current and upcoming releases. Community and contributor involvement is vital for successfully implementing all desired items for each release. We hope that the items listed below will inspire further engagement from the community to keep OpenEBS progressing and shipping exciting and valuable features.
+
+## OpenEBS Lean Roadmap
+
+OpenEBS follows a lean project management approach by splitting the development items into current, near term and future categories. We use GitHub [Projects](https://github.com/orgs/openebs/projects) and [Milestones](https://github.com/openebs/openebs/milestones) for tracking the feature development. This document is reviewed and updated based on the [monthly community call](https://github.com/openebs/openebs/tree/master/community#regular-monthly-product-meetings). 
+
+## Current
+
+These are some of the areas under active development planned to be completed within 3 releases. OpenEBS follows a monthly release cadence with a new release on 15 of every month.
```
Comment (muratkars) [inline ROADMAP.md]:
```suggestion
These are some of the areas under active development planned to be completed within three releases. OpenEBS follows a monthly release cadence with a new release on the 15th of every month. 
```

File: ROADMAP.md [hunk 10]
```diff
@@ -0,0 +1,58 @@
...
+
+## OpenEBS Lean Roadmap
+
+OpenEBS follows a lean project management approach by splitting the development items into current, near term and future categories. We use GitHub [Projects](https://github.com/orgs/openebs/projects) and [Milestones](https://github.com/openebs/openebs/milestones) for tracking the feature development. This document is reviewed and updated based on the [monthly community call](https://github.com/openebs/openebs/tree/master/community#regular-monthly-product-meetings). 
+
+## Current
+
+These are some of the areas under active development planned to be completed within 3 releases. OpenEBS follows a monthly release cadence with a new release on 15 of every month. 
+
+While the following are planned items, higher priority is given to usability and stability issues reported by the community. For the most up-to-date and issue plan and status check out the [release milestones](https://github.com/openebs/openebs/milestones).
```
Comment (muratkars) [inline ROADMAP.md]:
```suggestion
While the following are planned items, a higher priority is given to usability and stability issues reported by the community. For the most current issue plan and status, check out the [release milestones](https://github.com/openebs/openebs/milestones). 
```

File: ROADMAP.md [hunk 11]
```diff
@@ -0,0 +1,58 @@
...
+
+* [Jiva](https://github.com/orgs/openebs/projects/1)
+  * Optimizations to the replica rebuild process
+  * CSI Driver
+* [cStor](https://github.com/orgs/openebs/projects/9) 
+  * Support for local snapshot and clone via Velero
+  * New cStor API schema with automated Day 2 operations. 
+  * CSI Driver
+  * Support migration from SPC (older schema) to CSPC (new schema)
+* [Local PV - hostpath and device](https://github.com/orgs/openebs/projects/11)
```
Comment (muratkars) [inline ROADMAP.md]:
```suggestion
* [OpenEBS Local PV - hostpath and device](https://github.com/orgs/openebs/projects/11)
```

File: ROADMAP.md [hunk 12]
```diff
@@ -0,0 +1,58 @@
...
+  * Generating Metrics 
+  * Capacity based scheduling 
+* [Local PV - ZFS](https://github.com/orgs/openebs/projects/10)
```
Comment (muratkars) [inline ROADMAP.md]:
```suggestion
* [OpenEBS ZFS Local PV](https://github.com/orgs/openebs/projects/10)
```

File: ROADMAP.md [hunk 13]
```diff
@@ -0,0 +1,58 @@
...
+  * Generating metrics
+  * Integration and End-to-end tests
```
Comment (muratkars) [inline ROADMAP.md]:
```suggestion
  * Integration and end-to-end tests
```

File: ROADMAP.md [hunk 14]
```diff
@@ -0,0 +1,58 @@
...
+* ARM based builds for OpenEBS [#1295](https://github.com/openebs/openebs/issues/1295)
+
+## Near Term
+
+Typically the items under this category fall under 3 to 6 months roadmap. 
+
+There are several enhancements planned under each of the storage engines tracked under the respective GitHub Projects. The high level goals for each storage engine are as follows:
+* cStor with new schema (stable). [Project Tracker](https://github.com/orgs/openebs/projects/9)
+* NDM (stable). [Project Tracker](https://github.com/orgs/openebs/projects/2)
+* Local PV - ZFS (stable). [Project Tracker](https://github.com/orgs/openebs/projects/10)
```
Comment (muratkars) [inline ROADMAP.md]:
```suggestion
* OpenEBS ZFS Local PV (stable). [Project Tracker](https://github.com/orgs/openebs/projects/10)
```

File: ROADMAP.md [hunk 15]
```diff
@@ -0,0 +1,58 @@
...
+* Local PV - hostpath and device (stable). [Project Tracker](https://github.com/orgs/openebs/projects/11)
```
Comment (muratkars) [inline ROADMAP.md]:
```suggestion
* OpenEBS Local PV - hostpath and device (stable). [Project Tracker](https://github.com/orgs/openebs/projects/11)
```

File: ROADMAP.md [hunk 16]
```diff
@@ -0,0 +1,58 @@
...
+* MayaStor (beta)
```
Comment (muratkars) [inline ROADMAP.md]:
```suggestion
* Mayastor (beta)
```

File: ROADMAP.md [hunk 17]
```diff
@@ -0,0 +1,58 @@
...
+
+## Future
+
+As the name suggests this bucket contains items that are planned for future. Some times the items are related to adapting to the changes coming in the Kubernetes or other related projects. Some of the items currently planned include: 
+* MayaStor (stable)
```
Comment (muratkars) [inline ROADMAP.md]:
```suggestion
* Mayastor (stable)
```

File: ROADMAP.md [hunk 18]
```diff
@@ -0,0 +1,58 @@
...
+  * Generating metrics
+  * Integration and End-to-end tests
+  * Automated install and upgrades  
+* [NDM Enhancements](https://github.com/orgs/openebs/projects/2)
+  * Enhance the discovery probes to support partitions, lvms and so forth
+  * Support Prometheus exporter on block device metrics
+  * Add gRPC API layer around NDM capabilities
+* MayaStor 
+  * Replication across multiple nodes
+  * Integration and End-to-end tests
```
Comment (muratkars) [inline ROADMAP.md]:
```suggestion
  * Integration and end-to-end tests
```

File: contribute/process/release-management.md [hunk 19]
```diff
@@ -0,0 +1,71 @@
...
+## Release Tagging
+
+OpenEBS Components are released as container images with versioned tags. 
+
+OpenEBS components are spread across various repositories. Creating a release tag on the repository will trigger the build, integration tests and pushing of the docker image to [docker hub](https://hub.docker.com/u/openebs) and [quay](https://quay.io/organization/openebs/) container repositories. 
+
+The format of the release tag is either "Release-Name-RC1" or "Release-Name" depending on whether the tag is a release candidate or a release. (Example: 1.0.0-RC1 is a github release tag for OpenEBS release candidate build. 1.0.0 is the release tag that is created after the release criteria is satisfied by the release candidate builds.)
+
+Each of the repository has the automation scripts setup to push in the container images to docker and quay container repositories.
```
Comment (muratkars) [inline contribute/process/release-management.md]:
```suggestion
Each repository has the automation scripts setup to push in the container images to docker and quay container repositories. 
```

File: contribute/process/release-management.md [hunk 20]
```diff
@@ -0,0 +1,71 @@
...
+
+Once all the above tests are completed, a main release tagged images, helm and operator YAMLs are published.
+
+## Release Tagging
+
+OpenEBS Components are released as container images with versioned tags. 
+
+OpenEBS components are spread across various repositories. Creating a release tag on the repository will trigger the build, integration tests and pushing of the docker image to [docker hub](https://hub.docker.com/u/openebs) and [quay](https://quay.io/organization/openebs/) container repositories. 
+
+The format of the release tag is either "Release-Name-RC1" or "Release-Name" depending on whether the tag is a release candidate or a release. (Example: 1.0.0-RC1 is a github release tag for OpenEBS release candidate build. 1.0.0 is the release tag that is created after the release criteria is satisfied by the release candidate builds.)
```
Comment (muratkars) [inline contribute/process/release-management.md]:
```suggestion
The format of the release tag is either "Release-Name-RC1" or "Release-Name" depending on whether the tag is a release candidate or a release. (Example: 1.0.0-RC1 is a GitHub release tag for OpenEBS release candidate build. 1.0.0 is the release tag that is created after the release criteria are satisfied by the release candidate builds.)
```

**Comment by kmova** [comment]
Add linux-utils repo to the list of images to be built.

**Comment by kmova** [comment]
@muratkars @ChandanSagar - I have incorporated the comments. Can you please give this another go?

**Pull request merged**